docs/dev-guide/src/getting-started/running-a-node.md

```diff
@@ -0,0 +1,46 @@
+# Running a Creditcoin node
+
+## Running a development node
+
+Now that you've built a `creditcoin-node` from source, you can get a minimal development node running with:
+
+```bash
+./target/release/creditcoin-node --dev --mining-key 5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY --mining-threads 1
```

Comment:
Correct me if I'm wrong, but would we need to add the `--validator` flag to this command to get the example output?

Reply:
Ah so it's a little weird, but  `--dev` implies `--validator`. If you did switch to another chain (e.g. `--chain local`), instead of just using the `--dev` config, you would need to pass the `--validator`.